etd_app/mods_mapper.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import unicode_literals
+from bdrxml import mods
+
+
+class ModsMapper(object):
+
+    def __init__(self, thesis):
+        self.thesis = thesis
+
+    def get_mods(self):
+        return self._map_to_mods(self.thesis)
```

Comment:
yeah, true enough. I have half a mind to not make it a class at all - seems like it would work fine as a set of functions. But it's not helpful to be half-way in between. :) I'll update it to use self.thesis.